Examples/CaseStudies/SwiftUICaseStudies/01-GettingStarted-SharedState.swift

```diff
@@ -205,13 +206,9 @@ struct SharedStateCounterView: View {
       .frame(minWidth: 0, maxWidth: .infinity, minHeight: 0, maxHeight: .infinity, alignment: .top)
       .navigationBarTitle("Shared State Demo")
       .alert(
-        item: viewStore.binding(
-          get: { $0.alert.map(PrimeAlert.init(title:)) },
-          send: .alertDismissed
-        )
-      ) { alert in
-        SwiftUI.Alert(title: Text(alert.title))
-      }
+        self.store.scope(state: \.alert),
```

Comment:
this may be fixed in one of your other branches, but to get things building on Xcode 11.3:

```suggestion
        self.store.scope(state: { $0.alert }),
```